src/videostreamindex/VideoStreamDescription.ts

```diff
@@ -55,7 +55,7 @@ export default class VideoStreamDescription {
     descriptor.groupId = this.groupId;
     descriptor.framerate = this.maxFrameRate;
     descriptor.maxBitrateKbps =
-      this.disabledByUplinkPolicy || this.disabledByWebRTC ? 0 : this.maxBitrateKbps;
+      this.disabledByUplinkPolicy || this.disabledByUplinkPolicy ? 0 : this.maxBitrateKbps;
```

Comment:
`this.disabledByUplinkPolicy || this.disabledByUplinkPolicy` seems redundant.

Reply:
I'm sorry.  Somehow I messed up the merge. It's getting the older version back and my changes were not in this version. Will redo the merge...